source/_patterns/01-molecules/layouts/layouts.mustache

```diff
@@ -1,57 +1,85 @@
-<div class="layouts">
-  <div class="pad">
-    <h1>Standard one column layout</h1>
-    <p>All the content goes in 1 column.</p>
-  </div>
-  <div class="cf">
-    <div class=" pad-double bg--brown">
+<div class="layouts spacing--double">
+  <div>
```

Comment:
Yes, it creates the spacing [these selectors](https://github.com/adventistchurch/alps/blob/master/source/css/_modifier.spacing.scss#L126) create.